client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Site } from '../types';
+import SitePreviewPaneContent from './site-preview-pane-content';
+import SitePreviewPaneFooter from './site-preview-pane-footer';
+import SitePreviewPaneHeader from './site-preview-pane-header';
+import SitePreviewPaneTabs from './site-preview-pane-tabs';
+
+import './style.scss';
+
+interface Props {
+	selectedSite?: Site | undefined;
+	closeSitePreviewPane: () => void;
+}
+
+export default function SitePreviewPane( { selectedSite, closeSitePreviewPane }: Props ) {
+	if ( ! selectedSite ) {
+		return null;
+	}
+
+	return (
+		<div className="site-preview__pane">
+			<SitePreviewPaneHeader
+				selectedSite={ selectedSite }
```

Comment:
We could send to the header component just the 3 variables it needs via props:

- title
- url
- url_with_scheme
- icon
- closePreviewPane

That way, we make it totally independent.

Reply:
[dd13a8f](https://github.com/Automattic/wp-calypso/pull/87760/commits/dd13a8f96dc5685f01148cb227d610cf7f6c9de0)